test/test_2d.jl

```diff
@@ -127,6 +127,26 @@ end
       end
     end
   end
+
+  @testset "P4estMesh" begin
+    isdir(outdir) && rm(outdir, recursive=true)
+    run_trixi(joinpath("2d", "elixir_euler_nonperiodic_p4est.jl"), initial_refinement_level=1, maxiters=1)
```

Comment:
Have you tested this for higher refinement levels as well?

Reply:
Or does it make sense to include a specific test for it (or just make it the default here, e.g., `2`)?

Reply:
I tested it for levels 0, 1, and 2. But Trixi2Vtk doesn't use the level or the trees at all. The `node_coordinates` of each element are saved in the element container the same way as with `CurvedMesh`.

Reply:
Ah, right: level 1 is already one higher then the base level. In this case we can just leave it as it is, I think